mkdocs-project-dir/docs/howto.md

```diff
@@ -247,134 +314,184 @@ Host aristotle
 -----------------------------------------------------------------------------------------------------------------
-123454 2.00685 DI_m3      ccxxxxx      Eqw   10/13/2017 15:29:11                                    12 
-123456 2.00685 DI_m3      ccxxxxx      r     10/13/2017 15:29:11 Yorick@node-x02e-006               24 
-123457 2.00398 DI_m2      ucappka      qw    10/12/2017 14:42:12                                    1 
+123454 2.00685 DI_m3      ccxxxxx      Eqw   10/13/2017 15:29:11                                    12
+123456 2.00685 DI_m3      ccxxxxx      r     10/13/2017 15:29:11 Yorick@node-x02e-006               24
+123457 2.00398 DI_m2      ucappka      qw    10/12/2017 14:42:12                                    1
 ```
 
-This shows you the job ID, the numeric priority the scheduler has assigned to the job, the name you have given the job, your username, the state the job is in, the date and time it was submitted at (or started at, if it has begun), the head node of the job, the number of 'slots' it is taking up, and if it is an array job the last column shows the task ID.
+This shows you the job ID, the numeric priority the scheduler has assigned to
+the job, the name you have given the job, your username, the state the job is
+in, the date and time it was submitted at (or started at, if it has begun), the
+head node of the job, the number of 'slots' it is taking up, and if it is an
+array job the last column shows the task ID.
 
-The queue name (`Yorick` here) is generally not useful. The head node name (`node-x02e-006`) is useful - the `node-x` part tells you this is an X-type node. 
+The queue name (`Yorick` here) is generally not useful. The head node name
+(`node-x02e-006`) is useful - the `node-x` part tells you this is an X-type
+node.
+
+If you want to get more information on a particular job, note its job ID and
+then use the `-f` and `-j` flags to get full output about that job. Most of this
+information is not very useful.
 
-If you want to get more information on a particular job, note its job ID and then use the -f and -j flags to get full output about that job. Most of this information is not very useful.
 ```
 qstat -f -j 12345
 ```
 
 #### Job states
 
- `qw`: queueing, waiting
- `r`: running
- `Rq`: a pre-job check on a node failed and this job was put back in the queue
- `Rr`: this job was rescheduled but is now running on a new node
- `Eqw`: there was an error in this jobscript. This will not run.
- `t`: this job is being transferred
- `dr`: this job is being deleted
-<!--- dRr  --->
-<!--- hqw  --->
+* `qw`: queueing, waiting
+* `r`: running
+* `Rq`: a pre-job check on a node failed and this job was put back in the queue
+* `Rr`: this job was rescheduled but is now running on a new node
+* `Eqw`: there was an error in this jobscript. This will not run.
+* `t`: this job is being transferred
+* `dr`: this job is being deleted
```

Comment:
`d`: this job is being deleted 
(since it can be combined with any of the others)

Reply:
yes, edited in 2nd commit